packages/fuchsia_ctl/bin/main.dart

```diff
@@ -183,7 +183,7 @@ Future<OperationResult> test(
       return result;
     }
 
-    await server.serveRepo(repo.path, port: 54321);
+    await server.serveRepo(repo.path, port: 0);
```

Comment:
`0` means give me whatever?

Reply:
Yes - any available open port.  PM will then tell us what port it actually used.